### PR TITLE
i18n(fr): update `showcase.mdx`

### PR DESCRIPTION
**Pull request by HiDeoo**
#### What kind of changes does this PR include?

- Changes or translations of Starlight docs site content

#### Description

This PR updates `showcase.mdx` in french with the changes of #547.

It also translates the page title which was not translated.

### CODE REVIEW

File: docs/src/content/docs/fr/showcase.mdx [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+	<Card
+		title="CommandKit"
+		href="https://commandkit.underctrl.io"
+		thumbnail="commandkit.underctrl.io.png"
+	/>
+</FluidGrid>
+
+Voir tous les [dépôts publics de projets utilisant Starlight sur GitHub](https://github.com/withastro/starlight/network/dependents).
+
+## Modules d'extension communautaires
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/showcase.mdx:89]: An official reference for the translation of "plugins" https://www.culture.fr/franceterme/terme/INFO58

File: docs/src/content/docs/fr/showcase.mdx [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+<CardGrid>
+	<LinkCard
+		href="https://www.feelback.dev/blog/new-astro-starlight-integration/"
+		title="FeelBack"
+		description="Ajouter un système de retour d'expérience utilisateur à vos pages de documentation."
+	/>
+	<LinkCard
+		href="https://github.com/HiDeoo/starlight-blog"
+		title="starlight-blog"
+		description="Ajouter un blogue à votre site de documentation."
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/showcase.mdx:104]: An official reference for the translation of "blog" https://www.culture.fr/franceterme/terme/INFO870

**Review by dreyfus92 — APPROVED** [review]
Awesome job @HiDeoo, LGTM ✅

**Review by delucis — APPROVED** [review]
Thank you @HiDeoo — this looks great!

**Pull request merged**